src/test/spot.t.sol

```diff
@@ -0,0 +1,9 @@
+pragma solidity >=0.5.0;
+
+import "ds-test/test.sol";
+
+import {Spotter} from "../spot.sol";
+
+contract SpotterTest is DSTest {
```

Comment:
Is there any point having this stubbed?

Reply:
I guess it's pretty redundant. Reasoning was that it highlights lack of tests and suggests that we write some, but not a considered choice to include it - happy to remove.